client/src/UpdateForm.js

```diff
@@ -1,97 +0,0 @@
-import React, { Component } from "react";
-import "./App.css";
-import axios from "axios";
-
-const BASE_URL = "http://localhost:3005";
-
-class UpdateForm extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      name: "",
-      image_url: "",
-      scary: {
-        level: 1
-      }
-    };
-  }
-  handleChange = (event, key) => {
-    this.setState({
-      [key]: event.target.value
-    });
-  };
-
-  handleScaryLevel = event => {
-    const { scary } = this.state.scary;
-    const newScary = Object.assign({}, scary);
-    newScary.level = event.target.value;
-    this.setState({ scary: newScary });
-  };
-
-  handleClick = id => {
-    let newCreature = {
-      id: id,
-      name: this.state.name,
-      image_url: this.state.image_url,
-      scary: {
-        level: this.state.scary.level
-      }
-    };
-    console.log("Updating the list", newCreature);
-
-    axios.put(`${BASE_URL}/creatures/${id}`, newCreature).then(response => {
-      console.log("Updated", response);
-
-      this.props.updateState(response.data);
-    });
-  };
-
-  componentDidMount() {
-    console.log("Component mounted");
-    this.setState({
-      id: this.props.creature.id,
-      name: this.props.creature.name,
-      image_url: this.props.creature.image_url,
-      scary: this.props.creature.scary
-    });
```

Comment:
one cool way to write this would be to destructure, then use object shorthand: 
```
const { id, name, image_url, scary } = this.props
this.setState({
  id,
  name,
  image_url,
  scary
})
```